core/src/main/scala/caliban/schema/Types.scala

```diff
@@ -7,14 +7,19 @@ import scala.annotation.tailrec
 
 object Types {
 
-  /**
-   * Creates a new scalar type with the given name.
-   */
+  /** For bincompat */
   def makeScalar(
     name: String,
     description: Option[String] = None,
-    specifiedBy: Option[String] = None,
-    directives: Option[List[Directive]] = None
+    specifiedBy: Option[String] = None
+  ): __Type =
+    makeScalar(name, description, specifiedBy, None)
+
+  def makeScalar(
+    name: String,
+    description: Option[String],
+    specifiedBy: Option[String],
+    directives: Option[List[Directive]]
```

Comment:
So this change was source compatible but not binary compatible. I'm 50-50 whether I should revert this change and add this method as a filter instead

Reply:
Considering the changes I'm working on with the adapters, I think we may go 2.2 directly so maybe it's fine to keep those breaking changes.

Reply:
> How do we make Mima not fail when we know the next release will be a minor version release (in which case, we allow breaking changes)?

@ghostdogpr I  added a flag in `build.sbt` to disable CI breaking on binary incompatible changes. The good thing about this flag is that the plugin will still report the bincompat issues in the console, so we can always look at it when we're close to releasing to know what bin / source incompatible changes we introduced

Also, Since the next release will likely be 2.2.0, I reverted the changes to fix the bincompat issue in `makeScalar`.